src/style.css

```diff
@@ -1 +1,76 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nav_link {
+  height: 60px;
+  display: flex;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14.06px;
+  text-transform: uppercase;
+  text-decoration: none;
+  align-items: center;
+  color: #000;
+  position: relative;
```

Comment:
Pay attention to your CSS properties order.
https://9elements.com/css-rule-order/